src/app/work-item/work-item-board/work-item-board.component.scss

```diff
@@ -151,11 +148,15 @@
   overflow-y: scroll;
 }
 
+.clear {
+  clear: both;
+}
+
```

Comment:
No need to create another `.clear` class, please reuse the bootstrap CSS class for this.
It's defined in `node_modules/bootstrap/less/mixins/clearfix.less` and used in patternfly for your reference.